exploratory_analysis.ipynb

```diff
@@ -418,6 +363,29 @@
     "df.describe()"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "numerator     -0.036932\n",
+       "denominator    0.108359\n",
+       "raw_value      1.000000\n",
+       "Name: raw_value, dtype: float64"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df[['numerator', 'denominator', 'raw_value']].corr()['raw_value']"
```

Comment:
This is looking at the correlation with raw values across all the measures together so difficult for me to understand what the correlation means. Filtering for a single measure then computing correlation may be easier to interpret

Reply:
It is.  This is the first pass for me without having any information on how these values were calculated.  I  was just wondering whether whether there is any kind of relationship between these values.  I agree evaluating the data based on measures will make more sense.